src/tools/dartfmt.md

```diff
@@ -42,4 +42,13 @@ $ dartfmt --help
 
 {% comment %}
 [PENDING: Add info on commonly used options.]
+
+[PENDING: Advocate using this! Perhaps steal the first paragraph from
```

Comment:
I wrote this a while back; may serve as inspiration
https://flutter.dev/docs/development/tools/formatting

Reply:
Almost all of this content is applicable to Dart formatting, in general. Maybe we should have includes under site-shared? Or move this information to the Dart site?

I thought there might be a lint or other tool to help with trailing commas, but looking at the reported issues (https://github.com/dart-lang/dart_style/issues/753, https://github.com/dart-lang/sdk/issues/27086), it's apparently a hard problem to solve.